nima/webserver/webserver/src/main/java/io/helidon/nima/webserver/WebServer.java

```diff
@@ -233,9 +233,13 @@ public Builder config(Config config) {
                             connConfig.get("tcp-no-delay").asBoolean().ifPresent(socketOptionsBuilder::tcpNoDelay);
                         });
                     });
+            // Configure content encoding
+            config.get("content-encoding")
```

Comment:
Use `Config.get("content-encoding").as(ContentEncodingContext::create).ifPresent(this::contentEncodingContext);`

Reply:
done